creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -210,7 +242,17 @@ const _findSyncsForUser = (
         errors.push(
           `Error getting new or existing sync request for user ${wallet} and secondary ${secondary} - ${e.message}`
         )
+        continue
       }
+    } else if (syncMode === SYNC_MODES.SyncPrimaryFromSecondary) {
```

Comment:
nit - got slightly tripped up by "sync" terminology since we use it when it's either diffed or from scratch. you don't need to change this but just throwing out another suggestion, perhaps MergePrimaryAndSecondary?

Reply:
yeah fair enough - idk if that suggestion is best but it is better than current and i can't think of a better one so i'm gonna change to it